models/project/board.go

```diff
@@ -156,6 +156,15 @@ func NewBoard(ctx context.Context, board *Board) error {
 		return fmt.Errorf("bad color code: %s", board.Color)
 	}
 
+	var maxSorting int8
+	if _, err := db.GetEngine(ctx).Select("Max(sorting)").Table("project_board").
+		Where("project_id=?", board.ProjectID).Get(&maxSorting); err != nil {
+		return err
+	}
+	if maxSorting > 0 {
+		board.Sorting = maxSorting + 1
+	}
```

Comment:
1. It seems that we need a migration. 
For the existing project with only one default board, max sorting will always be 0.
2. sorting is int8, and will always +1, how to handle the overflow?

Reply:
This is columns number on a project, I don't think they should have too much columns. I think maybe we could have a max_number of columns for that but no migrations.

Reply:
max_number is necessary I think

Reply:
03562c7ff70c9d51e5140b1f38eb6c62ad40663b